include/great_expectations/plugins/expectations/expect_column_values_to_be_alphabetical.py

```diff
@@ -0,0 +1,268 @@
+                    "exact_match_out": False,
+                    "include_in_gallery": True,
+                    "in": {
+                        "column": "is_alphabetical_lowercase_reversed",
+                        "reverse": True,
+                        "mostly": 1.0,
+                    },
+                    "out": {
+                        "success": True,
+                        "unexpected_index_list": [],
+                        "unexpected_list": [],
+                    },
+                },
+                {
+                    "title": "positive_test_with_all_values_alphabetical_mixedcase",
+                    "exact_match_out": False,
+                    "include_in_gallery": True,
+                    "in": {"column": "is_alphabetical_mixedcase", "mostly": 1.0},
+                    "out": {
+                        "success": True,
+                        "unexpected_index_list": [],
+                        "unexpected_list": [],
+                    },
+                },
+                {
+                    "title": "negative_test_with_out_of_order_mixedcase",
+                    "exact_match_out": False,
+                    "include_in_gallery": True,
+                    "in": {"column": "out_of_order", "mostly": 1.0},
+                    "out": {
+                        "success": False,
+                        "unexpected_index_list": [2, 3, 7],
+                        "unexpected_list": ["up", "down", "east"],
+                    },
+                },
+            ],
+        }
+    ]
+
+    # This dictionary contains metadata for display in the public gallery
+    library_metadata = {
+        "maturity": "experimental",  # "experimental", "beta", or "production"
+        "tags": ["experimental"],  # Tags for this Expectation in the gallery
+        "contributors": [  # Github handles for all contributors to this Expectation.
+            "@sethdmay",
+            "@maximetokman",
+            "@Harriee02",  # Don't forget to add your github handle here!
+        ],
+        "package": "experimental_expectations",
+    }
+
+    # This is the id string of the Metric used by this Expectation.
+    # For most Expectations, it will be the same as the `condition_metric_name` defined in your Metric class above.
+    map_metric = "column_values.are_alphabetical"
+
+    # This is a list of parameter names that can affect whether the Expectation evaluates to True or False
+    # Please see {some doc} for more information about domain and success keys, and other arguments to Expectations
+    success_keys = ("mostly", "reverse")
+
+    # This dictionary contains default values for any parameters that should have default values
+    default_kwarg_values = {}
+
+if __name__ == "__main__":
+    diagnostics_report = ExpectColumnValuesToBeAlphabetical().run_diagnostics()
+    print(json.dumps(diagnostics_report, indent=2))
```

Comment:
Do we want to swap this over to the new `print_diagnostic_checklist()` pattern? Not a huge deal, not worth gating over.

Reply:
Totally! Great point.